Baikal/Controllers/clw_scene_controller.cpp

```diff
@@ -1214,23 +1258,31 @@ namespace Baikal
         }
     }
 
-    void ClwSceneController::WriteTexture(Texture const& texture, std::size_t data_offset, void* data) const
+    void ClwSceneController::WriteTexture(Texture const& texture, void* data, int mip_levels_offset) const
```

Comment:
std:;size_t instead of int?

Reply:
Ok, you're right, it will improve encapsulation.